test/index.html

```diff
@@ -7,8 +7,13 @@
 		<meta name="viewport" content="initial-scale=1.0, user-scalable=no" />
 
 		<!-- Leaflet JS / CSS -->
-		<script src="https://unpkg.com/leaflet@1.7/dist/leaflet-src.js"></script>
-		<link rel="stylesheet" href="https://unpkg.com/leaflet@1.7/dist/leaflet.css">
+		<script src="https://unpkg.com/leaflet@1.9/dist/leaflet-src.js"></script>
+		<link rel="stylesheet" href="https://unpkg.com/leaflet@1.9/dist/leaflet.css">
+
+		<!-- Leaflet-MarkerCluster -->
+		<script src="https://unpkg.com/leaflet.markercluster@1.4.1/dist/leaflet.markercluster-src.js"></script>
+		<link rel="stylesheet" href="https://unpkg.com/leaflet.markercluster@1.4.1/dist/MarkerCluster.css">
+		<link rel="stylesheet" href="https://unpkg.com/leaflet.markercluster@1.4.1/dist/MarkerCluster.Default.css">
 
```

Comment:
@siimaoo for your tests, here is a complete example of integration with leaflet v1.9 and leaflet.markercluster

Reply:
@Raruto sorry, I running the test on the wrong file! Now, everything is good!

Reply:
Is working with cluster, draggable markers and popups! Nice!